### ISSUE TITLE
Why --tabs=4 for less?

### ISSUE THREAD

**Issue by Blaisorblade** [so-fancy/diff-so-fancy#174]
As per subject. I'd like to use this tool without entering a religion war on the size of tabs 😄.
- Either `--tabs=4` is used only because the authors prefer so for other reasons, and is not actually needed. and then the README could be improved;
- Or diff-so-fancy actually relies on that, and then it might not be compatible with some codebases I'm using that rely on tabs being 8 space wide.

My current config:
```
$ git config --global core.pager
less -+$LESS -FRX -i
```

Config I plan to try out:
```
git config --global core.pager='diff-so-fancy |less -+$LESS -FRX -i'
```

**Comment by scottchiefbaker** [comment]
I don't have a `$LESS` environment variable, should I?

**Comment by Blaisorblade** [comment]
Uh, neither have I — but it can be set by `git` before invoking the pager.

But my main point was another:

TL; DR. Do you expect `diff-so-fancy` relies on special `less` options, other than `-R`?

I'd like to keep the existing less command line (with my addition of `-i`) and just prepend `diff-so-fancy |`, if that's possible (which it seems, but I haven't done extensive testing).

----

Regarding "what the heck is that setting I have", it turns out it might be outdated — it seems that `less -+$LESS -RSX` was the old recommended default, but no more. http://stackoverflow.com/a/12352224/53974

**Comment by OJFord** [comment]
This ties in with @stevemao's desire to not document "how to configure git/shell".

I assume DSF doesn't rely on `--tabs=4` (if so we should open a new issue "Don't rely on ..."!) - this can probably be changed for something more suggestive, like:
> the output of `diff-so-fancy` should then be piped to your existing pager, or if you don't have one, you might choose to use something like `less --tabs=4 -RFX`

**Comment by scottchiefbaker** [comment]
I don't have "an existing pager" other than vanilla less. I think your wording is reasonable, with a minor tweak:

> the output of `diff-so-fancy` should then be piped to your existing pager, or if you don't have one, we recommend `less --tabs=4 -RFX`

**Comment by OJFord** [comment]
> I don't have "an existing pager"

Right, which is why a suggestion is helpful, but it's not unreasonable to think that someone might. I used to have it explicitly set to `less`, partly for the sake of being explicit, and partly to remind myself that it _wasn't_ `$EDITOR`.
